fileio/ft_read_event.m

```diff
@@ -492,15 +492,33 @@
       hdr = ft_read_header(filename);
     end
     % the following applies to Biosemi data that is stored in the gdf format
-    statusindx = find(strcmp(hdr.label, 'STATUS'));
-    if length(statusindx)==1
+    chanindx = find(strcmp(hdr.label, 'STATUS'));
```

Comment:
`chanindx` should only be overwritten with the STATUS channel index when empty, as it can also be specified explicitly by the user.

Reply:
good point, I fixed this